doc/active_docs/account_management_api.json

```diff
@@ -903,7 +903,7 @@
                   },
                   "key": {
                     "type": "string",
-                    "description": "app_key to be added"
+                    "description": "app_key to be added.\n\n**(Only alphanumeric characters `[0-9, a-z, A-Z]`, hyphen-minus (`-`), between 5 and 256 characters long. No spaces allowed).**"
```

Comment:
App Key has the least amount of restrictions, and it allows almost any punctuation symbols. So, maybe we need to rectify that.